src/lib/party.ts

```diff
@@ -231,7 +242,7 @@ let Permissions = {
   }),
 };
 
-const getDefaultTokenId = () => Field.one;
+const getDefaultTokenId = () => Ledger.fieldToBase58(Field.one);
```

Comment:
For dealing with variables, it's probably best / most convenient to have tokens represented as `Field` most of the time in snarkyjs, and only using the string versions when doing IO. At least that's what we generally do with other values, like public/private keys, hashes, balances etc

Reply:
Which is to say I'd probably revert this change (but don't do it if it's less convenient that way)

Reply:
Good call, fixed!